arrow-meta/build.gradle

```
@@ -10,28 +10,28 @@ apply from: "$SUB_PROJECT"
 apply from: "$DOC_CREATION"
 
 dependencies {
-    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$KOTLIN_VERSION"
-    compile project(':arrow-annotations')
-    compile "me.eugeniomarletti.kotlin.metadata:kotlin-metadata:$KOTLIN_METADATA_VERSION"
-    compile("com.squareup:kotlinpoet:$KOTLIN_POET_VERSION") {
+    api "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$KOTLIN_VERSION"
+    api project(':arrow-annotations')
+    api "me.eugeniomarletti.kotlin.metadata:kotlin-metadata:$KOTLIN_METADATA_VERSION"
+    api("com.squareup:kotlinpoet:$KOTLIN_POET_VERSION") {
```

Those should stay within arrow-meta only AFAIK

```suggestion
    implementation "me.eugeniomarletti.kotlin.metadata:kotlin-metadata:$KOTLIN_METADATA_VERSION"
    implementation("com.squareup:kotlinpoet:$KOTLIN_POET_VERSION") {
```

---

I guess they are necessary to use `arrow-meta`